### DISCUSSION TITLE
Debug mode?

### DISCUSSION THREAD

**Discussion by netmgr** [erichelgeson/BlueSCSI-v1#213] [Q&A] [answered]
Hi Folks
I see that the code has reference to a debug mode, do I need to compile the code myself to enable this or is there an SD card command file I can use?

**Comment by erichelgeson** [comment, marked as answer]
This is a compile time option - you can find the setting and values here https://github.com/erichelgeson/BlueSCSI/blob/main/src/BlueSCSI.cpp#L42-L44
